packages/react/toast/src/Toast.tsx

```diff
@@ -794,21 +778,47 @@ const ToastClose = React.forwardRef<ToastCloseElement, ToastCloseProps>(
   (props: ScopedProps<ToastCloseProps>, forwardedRef) => {
     const { __scopeToast, ...closeProps } = props;
     const interactiveContext = useToastInteractiveContext(CLOSE_NAME, __scopeToast);
-    return interactiveContext.isInteractive ? (
-      <Primitive.button
-        type="button"
-        {...closeProps}
-        ref={forwardedRef}
-        onClick={composeEventHandlers(props.onClick, interactiveContext.onClose)}
-      />
-    ) : null;
+
+    return (
+      <ToastAnnounceExclude asChild>
```

Comment:
What happens for this one as it has no `altText`?
I can imagine it will read the `textContent` (say "close") but what if it's a close icon with an `aria-label`?